### PR TITLE
Add Piwik tracking support

### PR DESCRIPTION
**Pull request by xenithorb**

### CODE REVIEW

File: templates/include/piwik.html [hunk 1]
```diff
@@ -0,0 +1,19 @@
+{% if PIWIK_URL and PIWIK_SITE_ID %} 
+
```
Comment (ingwinlu) [inline templates/include/piwik.html]: remove line

File: templates/include/piwik.html [hunk 2]
```diff
@@ -0,0 +1,19 @@
...
+    var u="{{ PIWIK_URL }}";
+    _paq.push(['setTrackerUrl', u+'piwik.php']);
+    _paq.push(['setSiteId', {{ PIWIK_SITE_ID }}]);
+    var d=document, g=d.createElement('script'), s=d.getElementsByTagName('script')[0];
+    g.type='text/javascript'; g.async=true; g.defer=true; g.src=u+'piwik.js'; s.parentNode.insertBefore(g,s);
+  })();
+</script>
+<noscript><p><img src="{{ PIWIK_URL }}/piwik.php?idsite={{ PIWIK_SITE_ID }}" style="border:0;" alt="" /></p></noscript>
+<!-- End Piwik Code -->
+
```
Comment (ingwinlu) [inline templates/include/piwik.html]: remove line

**Comment by ingwinlu** [comment]
please incorporate the suggested changes and squash your commits

**Comment by xenithorb** [comment]
Updated with requested changes

**Comment by ingwinlu** [comment]
please review your commit message; see http://chris.beams.io/posts/git-commit/ or https://github.com/erlang/otp/wiki/Writing-good-commit-messages for input.
having lines like 'Missed a variable for PIWIK_SITE_ID' or 'Removed whitespace lines' makes no sense.

for future pull requests also consider not opening pr's from your master branch.

**Comment by xenithorb** [comment]
Updated

**Comment by ingwinlu** [comment]
Another missing thing is a description of the two new settings in `README.md`.

the template also adds +<noscript><p></p></noscript> which is just there to track people disabling js, can we just drop that?

**Comment by xenithorb** [comment]
Now you're just screwing with me

**Comment by ingwinlu** [comment]
No, Readme.md has a list of all settings. you are introducing new settings. it is missing in the pr.

**Comment by xenithorb** [comment]
I would have taken less time to add it yourself than to continue discussing it here if it mattered that much to you. Why bother contributing if we're just going to keep going back and forth. I've canceled my PR and filed a feature request. Maybe other people will vote on it. Take care

**Comment by ingwinlu** [comment]
It also served as a reminder to myself on why I haven't simply pressed the merge button.

I am sorry you feel like copy pasting a code snipped wasted so much of your time that you have to act that way

**Pull request closed without merging**